packages/player/src/index.ts

```diff
@@ -25,6 +25,8 @@ type Options = {
 export function getMediaElement(): HTMLMediaElement | null {
   const { activePlayer: player } = playerState;
 
+  console.log('activePlayer', { player });
```

Comment:
Do we want to log here?

Reply:
Nein. Removed in [dde517b](https://github.com/tidal-music/tidal-sdk-web/pull/23/commits/dde517b9c4fdc6e3ecce96b3ea025928c168f903)